<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Matus &amp; Michal Tomlein</author>
 <class>ImportCsvDialogue</class>
 <widget class="QDialog" name="ImportCsvDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import CSV - Leaklog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/resources.qrc">
    <normaloff>:/images/images/import.png</normaloff>:/images/images/import.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>12</number>
   </property>
   <property name="spacing">
    <number>12</number>
   </property>
   <item row="0" column="2">
    <widget class="QLabel" name="lbl_encoding">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Encoding:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QComboBox" name="cb_encoding"/>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="lbl_separator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Field separator:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QComboBox" name="cb_separator"/>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="lbl_skip_lines">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Number of lines to skip:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QSpinBox" name="spb_skip_lines"/>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="btn_reload">
     <property name="text">
      <string>Reload</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeWidget" name="trw_columns">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Index</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2" colspan="10">
    <widget class="QTableWidget" name="tw_content">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="9">
    <widget class="QDialogButtonBox" name="id_bb">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Table:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cb_table"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QToolButton" name="add_linked_table_btn">
     <property name="text">
      <string>Add linked table</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <resources>
  <include location="../rc/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>id_bb</sender>
   <signal>accepted()</signal>
   <receiver>ImportCsvDialogue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>id_bb</sender>
   <signal>rejected()</signal>
   <receiver>ImportCsvDialogue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
